datasets/cell-painting-image-collection.yaml

```diff
@@ -0,0 +1,46 @@
+Name: Cell Painting Image Collection
+Description: 
+  The Cell Painting Image Collection is a collection of freely 
+  downloadable microscopy image sets. Cell Painting is an 
+  unbiased high throughput imaging assay used to analyze 
+  perturbations in cell models. In addition to the images 
+  themselves, each set includes a description of the biological 
+  application and some type of "ground truth" (expected results). 
+  Researchers are encouraged to use these image sets as reference 
+  points when developing, testing, and publishing new image 
+  analysis algorithms for the life sciences. We hope that the 
+  this data set will lead to a better understanding of which 
+  methods are best for various biological image analysis 
+  applications.  
+Contact: imagingadmin@broadinstitute.org
+UpdateFrequency: unregularly 
+Tags:
+  - high throughput imaging
+  - cell imaging
+  - microscopy 
+  - Cell Painting
+  - fluorescence imaging
+License: CC0 1.0 Universal (CC0 1.0) Public Domain Dedication (datasets Bioactives-BBBC022-Gustafsdottir, CDRPBIO-BBBC036-Bray, LUAD-BBBC041-Caicedo) and CC BY 4.0 (TA-ORF-BBBC037-Rohban)
```

Comment:
@cells2numbers It might be ok to go with CC0 1.0 for TA-ORF-BBBC037-Rohban as well. I know I had previously suggested CC BY 4.0 to be compatible with the paper in which it was first published, but that license is for the paper, and doesn't necessarily need to apply to the dataset. I'll discuss this by email.